package-lock.json

```diff
@@ -2053,16 +2048,6 @@
         "lazy-cache": "^1.0.3"
       }
     },
-    "chalk": {
```

Comment:
Could you try to run `npm dedupe` or deduplicate `chalk` and `ansi-style`? It seems they are duplicated many times.